sync.py

```diff
@@ -111,26 +115,54 @@ def sync():
 
     # make a set of foxpass users that should be in duo. they must be active and if a group is
     # specified, they must be in that group
-    foxpass_email_set = set()
+    foxpass_sync_set = list()
```

Comment:
why switch this from a set to a list? seems to be doing the same thing, but a set is more efficient.